src/overlay/test/OverlayTestUtils.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+// Copyright 2022 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "main/Application.h"
```

Comment:
No need for this include here - you can forward declare Application, and include it in the cpp file